openstack/compute/v2/extensions/availabilityzones/results.go

```diff
@@ -1,8 +1,85 @@
 package availabilityzones
 
-// ServerAvailabilityZoneExt is an extension to the base Server result which
-// includes the Availability Zone information.
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+// ServerExt is an extension to the base Server object
 type ServerAvailabilityZoneExt struct {
 	// AvailabilityZone is the availabilty zone the server is in.
 	AvailabilityZone string `json:"OS-EXT-AZ:availability_zone"`
 }
+
+// UnmarshalJSON to override default
+func (r *ServerAvailabilityZoneExt) UnmarshalJSON(b []byte) error {
+	return nil
+}
```

Comment:
This was removed sometime after #327 was opened. Can you remove this function?

Reply:
No problem

Reply:
Thank you for feedback!

Reply:
@jtopjian is it okay to leave `ComputeAvailabilityZonePage` name instead `OSAvailabilityZonePage` ? The problem is that we've already used `AvailabilityZonePage` name in `openstack/sharedfilesystems/v2/availabilityzones/results.go`.

Reply:
You can re-use the name `AvailabilityZonePage`. The other `AvailabilityZonePage` is in another Go package and, unless I'm mistaken, there won't be a name collision.